--- a/ipheth-pair/90-iphone-tether.rules
+++ b/ipheth-pair/90-iphone-tether.rules
@@ -2,6 +2,6 @@
 ATTR{idVendor}!="05ac", GOTO="ipheth_rules_end"
 
 # Execute pairing program when appropriate
-ACTION=="add", SUBSYSTEM=="net", ENV{ID_USB_DRIVER}=="ipheth", SYMLINK+="iphone", RUN+="/usr/local/bin/ipheth-pair"
+ACTION=="add", SUBSYSTEM=="net", ENV{ID_USB_DRIVER}=="ipheth", SYMLINK+="iphone", RUN+="ipheth-pair"
 
 LABEL="ipheth_rules_end"
