--- a/ipheth-pair/ipheth-pair.c
+++ b/ipheth-pair/ipheth-pair.c
@@ -81,7 +81,7 @@
 	    char *device_name = NULL;
 	    rv = iphone_device_new(&device, devices[i]);
 	    if (rv == 0) {
-		rv = lockdownd_client_new(device, &client);
+		rv = lockdownd_client_new_with_handshake(device, &client, "ipheth-pair");
 		if (rv == 0) {
 		    rv = lockdownd_get_device_name(client, &device_name);
 		    lockdownd_client_free(client);
@@ -102,7 +102,7 @@
 	fprintf(stderr, "%s: cannot get %s device\n", argv[0], uuid ? uuid : "default");
 	return -1;
     }
-    rv = lockdownd_client_new(device, &client);
+    rv = lockdownd_client_new_with_handshake(device, &client, "ipheth-pair");
     if (rv) {
 	fprintf(stderr, "%s: cannot get lockdown\n", argv[0]);
 	iphone_device_free(device);
